File: src/2270. Number of Ways to Split Array.py
class Solution:
    def waysToSplitArray(self, nums: List[int]) -> int:
        left = sum(nums)
        ans = 0
        right = 0
        for i in range(len(nums) - 1):
            right += nums[i]
            left -= nums[i]
            if right >= left:
                ans += 1
        return ans

File: src/2270. Number of Ways to Split Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
function waysToSplitArray(nums) {
  let left = nums.reduce((acc, v) => acc + v, 0)
  let ans = 0
  let right = 0
  for (let i = 0; i < nums.length - 1; i++) {
    right += nums[i]
    left -= nums[i]
    if (right >= left) {
      ans++
    }
  }
  return ans
}
